readinglist/authentication.py

```diff
@@ -87,6 +87,7 @@ def permits(self, context, principals, permission):
         Authenticated users only are allowed.
         """
         PERMISSIONS = {
+            'batch': Authenticated,
```

Comment:
Indeed, why not leave this endpoint open and rely on sub-requests permission checks...